src/filters/index.js

```diff
@@ -61,7 +61,7 @@ var digitsRE = /(\d{3})(?=\d)/g
 exports.currency = function (value, currency) {
   value = parseFloat(value)
   if (!isFinite(value) || (!value && value !== 0)) return ''
-  currency = currency || '$'
+  currency = currency === false ? '' : currency || '$'
```

Comment:
Looking at #1131 , it seems we should just check if `currency` is undefined or null.
So something like `currency = currency != null ? currency : '$'`